package.json

```diff
@@ -29,7 +29,7 @@
     "ember-cli-lodash-subset": "2.0.1",
     "ember-cli-preprocess-registry": "^3.1.0",
     "ember-cli-version-checker": "^2.1.0",
-    "fastboot": "^1.1.3",
+    "fastboot": "github:rondale-sc/fastboot#utilize-rehydration-serialization-from-glimmer",
```

Comment:
Just notating that this will need to be bumped to a release version before landing...